orientation_bridge:
  ros__parameters:
    base_frame: "Orca/camera_front"
    map_frame: "Orca/camera_front"
    marker_frame: valve_marker_0
    marker_pose_topic: /aruco_detector/marker_pose
    publish_tf: false
    use_tf: false

    # Sliding window
    win.size: 15
    win.age_sec: 0.6
    win.huber_deg: 3.0
    win.gate_deg: 10.0
    win.use_mad: true

    pub_rate_hz: 30.0
    rpy_std_fallback_deg: 1.5
